src/SSHLibrary/pythonclient.py

```diff
@@ -30,7 +30,10 @@
 # There doesn't seem to be a simpler way to increase banner timeout
 def _custom_start_client(self, event=None, timeout=None):
     self.banner_timeout = 45
-    self._orig_start_client(event, timeout=timeout)
+    if timeout is not None:
+        self._orig_start_client(event, timeout=timeout)
+    else:
+        self._orig_start_client(event)
 
```

Comment:
Looked closer at pythonclient.py, and indeed _custom_start_client is expected to pass all arguments and keyword arguments as-is to _orig_start_client. So my latter code snipped should be used I think.